### ISSUE TITLE
bTrem and fTrem beam/slash miscalculation

### ISSUE THREAD

**Issue by jacekiwaszko1** [rism-ch/verovio#1448]
It looks like beams and slashes are miscalculated in rendering of `<bTrem>`s and `<fTrem>`s.

both folowing examples are encoded as eight note tremolos, both have two slashes/beams, as if it was sixteenth note tremolo:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="https://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="https://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="4.0.0">
    <music>
        <body>
            <mdiv>
                <score>
                    <scoreDef>
                        <staffGrp>
                            <staffDef n="1" lines="5">
                                <clef shape="G" line="2" />
                            </staffDef>
                        </staffGrp>
                    </scoreDef>
                    <section>
                        <measure>
                            <staff n="1">
                                <layer n="1">
                                    <bTrem unitdur="8">
                                        <note dur="2" oct="4" pname="c" accid.ges="n" />
                                    </bTrem>
                                </layer>
                            </staff>
                        </measure>
                    </section>
                </score>
            </mdiv>
        </body>
    </music>
</mei>
```

current rendering:

![btrem1](https://user-images.githubusercontent.com/29225574/81150008-81d41580-8f7f-11ea-9156-5d70c65b1cae.png)

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="https://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="https://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="4.0.0">
    <music>
        <body>
            <mdiv>
                <score>
                    <scoreDef>
                        <staffGrp>
                            <staffDef n="1" lines="5">
                                <clef shape="G" line="2" />
                            </staffDef>
                        </staffGrp>
                    </scoreDef>
                    <section>
                        <measure>
                            <staff n="1">
                                <layer n="1">
                                    <fTrem beams="1" unitdur="8">
                                        <note dur="2" oct="4" pname="c" accid.ges="n" />
                                        <note dur="2" oct="4" pname="d" accid.ges="n" />
                                    </fTrem>
                                </layer>
                            </staff>
                        </measure>
                    </section>
                </score>
            </mdiv>
        </body>
    </music>
</mei>
```

current rendering:

![ftrem1](https://user-images.githubusercontent.com/29225574/81150132-b3e57780-8f7f-11ea-9ffc-5a6965de4682.png)

Related to issue [humdrum-tools/verovio-humdrum-viewer/issues/302](https://github.com/humdrum-tools/verovio-humdrum-viewer/issues/302)

**Comment by rettinghaus** [comment]
Addressed in #1449 
The `<fTrem>` error came with 04aa712b8c928346562410c00720a6f1612d3605

Note, that your first encoding is incomplete. The guidelines say: 
> The __@stem.mod__ attribute must also be explicity set on the child `<note>` or `<chord>` element for a complete visual representation.

**Comment by jacekiwaszko1** [comment]
>Note, that your first encoding is incomplete. The guidelines say:
>
>>The @stem.mod attribute must also be explicity set on the child <note> or <chord> element for a complete visual representation.

Thanks for the advice! With `@stem.mod` rendering is correct:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="https://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="https://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="4.0.0">
    <music>
        <body>
            <mdiv>
                <score>
                    <scoreDef>
                        <staffGrp>
                            <staffDef n="1" lines="5">
                                <clef shape="G" line="2" />
                            </staffDef>
                        </staffGrp>
                    </scoreDef>
                    <section>
                        <measure>
                            <staff n="1">
                                <layer n="1">
                                    <bTrem unitdur="8">
                                        <note dur="2" oct="4" pname="c" accid.ges="n" stem.mod="1slash" />
                                    </bTrem>
                                </layer>
                            </staff>
                        </measure>
                    </section>
                </score>
            </mdiv>
        </body>
    </music>
</mei>
```

![btrem2](https://user-images.githubusercontent.com/29225574/81160050-e98f5e00-8f89-11ea-8e6f-aafb52c2c78f.png)
